.github/workflows/eslint-ci.yml

```diff
@@ -14,7 +14,14 @@ jobs:
       - uses: actions/setup-node@v2.5.1
         with:
           cache: 'pnpm'
+
+      - name: Turborepo local server
+        uses: felixmosh/turborepo-gh-artifacts@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          server-token: ${{ secrets.TURBO_SERVER_TOKEN }}
```

Comment:
What is this token supposed to look like?  I'm reading the docs and I don't totally understand the point of it.

Reply:
I've added `TURBO_SERVER_TOKEN` to the repo.